boot/configure.ml

```diff
@@ -70,6 +74,10 @@ let () =
       , Arg.String (set_dir datadir)
       , "DIR where files for the share_root section are installed for the default build \
          context" )
+    ; ( "--toggles"
```

Comment:
I think "--enable-toggles" would be a more accurate name for this argument.

Reply:
As discussed in the meeting, the `--help` argument should be useful. In particular, we want:

1. a separate command line flag (or argument) for every toggle
2. a description of what this flag does
3. a note that states that this flag is experimental and packagers should not rely on it

Reply:
> As discussed in the meeting, the `--help` argument should be useful. In particular, we want:
> 
>     1. a separate command line flag (or argument) for every toggle
> 
>     2. a description of what this flag does
> 
>     3. a note that states that this flag is experimental and packagers should not rely on it

Done!